libs/application/templates/social-insurance-administration/medical-and-rehabilitation-payments/src/forms/Prerequisites/externalDataSubSection.ts

```diff
@@ -40,39 +40,50 @@ export const externalDataSubSection = buildSubSection({
       }),
       dataProviders: [
         buildDataProviderItem({
-          // TODO: Update text!
           provider: NationalRegistryUserApi,
-          title: socialInsuranceAdministrationMessage.pre.skraInformationTitle,
-          subTitle: '',
+          title:
+            medicalAndRehabilitationPaymentsFormMessage.pre
+              .skraInformationTitle,
+          subTitle:
+            medicalAndRehabilitationPaymentsFormMessage.pre
+              .skraInformationDescription,
         }),
         buildDataProviderItem({
-          // TODO: Update text?
           provider: NationalRegistrySpouseApi,
           title: '',
         }),
         buildDataProviderItem({
           provider: UserProfileApi,
-          title: socialInsuranceAdministrationMessage.pre.contactInfoTitle,
+          title:
+            medicalAndRehabilitationPaymentsFormMessage.pre.contactInfoTitle,
```

Comment:
Use `contactInfoTitle` in core
```suggestion
          title: socialInsuranceAdministrationMessage.pre.contactInfoTitle,
```

Reply:
Used core msg